lib/no_retry_jobs.rb

```diff
@@ -0,0 +1,9 @@
+class NoRetryJobs
+  def call(_worker, msg, queue)
+    return unless queue == 'idv'
```

Comment:
This prevents jobs that aren't in the idv queue from running all. I think you want this guard clause after line 5 below.

Reply:
Derp! Good catch.

Reply:
Fixed here: https://github.com/18F/identity-idp/pull/1707/commits/eaa9358dcff65f11bf7afac43d58efef7987991e